packages/@webex/plugin-meetings/src/reachability/index.ts

```diff
@@ -425,7 +431,26 @@ export default class Reachability extends EventsScope {
           }
           break;
         case 'latencyInMilliseconds':
-          output.latencyInMilliseconds = value.toString();
+          if (value !== undefined && value !== null) {
+            output.latencyInMilliseconds = value.toString();
+          }
+          break;
+        // Transforming serverIp to serverIPs
+        case 'details':
+          if (Array.isArray(value)) {
+            output.details = value.map((subnet) => {
+              // Use serverIp if present, otherwise use serverIps
```

Comment:
our `details` array will never contain `serverIps` so why do we have this comment and code that tries to use it